shared/rate_limits/__init__.py

```diff
@@ -39,33 +55,49 @@ def determine_entity_redis_key(
     else:
         auth_info: AdapterAuthInformation = get_adapter_auth_information(owner=owner)
 
-    if auth_info.get("selected_installation_info"):
-        return f"{auth_info.get("selected_installation_info").get("app_id")}_{auth_info.get("selected_installation_info").get("installation_id")}"
-    # How to treat gh app fallbacks?
-    else:
-        return str(auth_info.get("token_owner").ownerid)
+    if (
+        auth_info
+        and auth_info.get("token")
+        and auth_info.get("token").get("entity_name")
+    ):
+        return auth_info.get("token").get("entity_name")
 
 
-# Ensure service = "gh" before using
 def determine_if_entity_is_rate_limited(redis_connection: Redis, key_name: str) -> bool:
     """
     This function will determine if a customer is rate limited. It will
     return true if the record exists, false otherwise.
-    This will be used by API and Worker
+    This will be used by API and Worker and should only be used for github git instances.
     """
-    return True if redis_connection.get(f"rate_limited_entity_{key_name}") else False
+    try:
+        return redis_connection.exists(f"rate_limited_entity_{key_name}")
```

Comment:
[nit] it might be a good idea to extract `rate_limited_entity_` into a constant.

```python
RATE_LIMIT_REDIX_KEY_PREFIX="rate_limited_entity_"
...
return redis_connection.exists(f"{RATE_LIMIT_REDIX_KEY_PREFIX}{key_name}")
```